src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -715,7 +715,8 @@ private void NotifyFeaturesAboutStateChange(DatabaseRecord record, long index)
                 Debug.Assert(string.Equals(Name, record.DatabaseName, StringComparison.OrdinalIgnoreCase),
                     $"{Name} != {record.DatabaseName}");
 
-                if (LastDatabaseRecordIndex >= index)
+                // index and LastDatabaseRecordIndex could have equal values when we transit from/to passive and want to update the tasks. 
+                if (LastDatabaseRecordIndex > index)
```

Comment:
Updating the cluster topology increase the raft index, so we had no problem in the case of not replicating *to*. The problem was to handle this on the passive node.